static/src/javascripts/projects/common/modules/experiments/tests/liveblog-epic-design-test.js

```diff
@@ -21,7 +21,7 @@ const copyGlobal: AcquisitionsEpicTemplateCopy = {
 
 const copyElectionNonUS: AcquisitionsEpicTemplateCopy = {
     paragraphs: [
-        'Four more years of Donald Trump is a real possibility. America faces an epic choice in November and the result of the presidential election will have global repercussions for democracy, progress and solidarity for generations. Transatlantic ties, superpower relations and the climate emergency are all in the balance.',
+        'Four more years of Donald Trump  would have serious consequences for the world. America faces an epic choice in November and the result of the presidential election will have global repercussions for democracy, progress and solidarity for generations. Transatlantic ties, superpower relations and the climate emergency are all in the balance.',
```

Comment:
the screenshot looks fine, but I think there's a double space here?

Reply:
good spot!